glustercli/cmd/root.go

```diff
@@ -13,7 +13,8 @@ var RootCmd = &cobra.Command{
 	Use:   "glustercli",
 	Short: "Gluster Console Manager (command line utility)",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		err := logging.Init(flagLogDir, flagLogFile, flagLogLevel, false)
+		//err := logging.Init(flagLogDir, flagLogFile, flagLogLevel, false)
+		err := logging.Init("", "stdout", flagLogLevel, false)
```

Comment:
We don't have to use `logging.Init()`. If we use logrus directly, we can customize the formatter too,

Reply:
I was intentionally left it,  wanted to make use of current function